### PR TITLE
EWL-4526:  Date and social share icons needs to line up on the news article page

### PR DESCRIPTION
**Pull request by designerneil**
## Ticket(s)

**Jira Ticket**
- [EWL-4526: SG2 |  News article: Date and social share icons needs to line up](https://issues.ama-assn.org/browse/EWL-4526)

## Description
UX has asked that the social icons line up with the date on the news article page. In addition, they also asked that the date be an h3

## To Test
- [ ] `gulp serve`
- [ ] in the SG2 menu click on page > [news](http://localhost:3000/?p=pages-news)
- [ ] observe the date as h3 
- [ ] observe the date and social icons lining up horizontally
- [ ] run` gulp backstop test` note any failures

## Visual Regressions
Added news test to backstop.json

## Relevant Screenshots/GIFs
screen shot 2018-02-09 at 3 21 01 pm

## Remaining Tasks
N/A

## Additional Notes
N/A

---

[Guidelines for Contribution](CONTRIBUTING.md)
[SG2 Standards](ama-style-guide-2/docs/standards.md)

### CODE REVIEW

**Review by labbydev — APPROVED** [review]
LGTM! Please resolve the merge conflicts and make sure that when you do you're changing the correct element.

**Comment by labbydev** [comment]
Sorry @designerneil still some merge conflicts.

**Comment by designerneil** [comment]
@labbydev yeah i just the conflict. it's hard to keep up here

File: styleguide/source/_patterns/03-organisms/article-header.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
+---
+el: ".ama__header--article"
```
Comment (labbydev) [inline styleguide/source/_patterns/03-organisms/article-header.md]: This file should be deleted.

**Review by labbydev — CHANGES_REQUESTED** [review]
`article-header.twig` should also be deleted

**Review by labbydev — APPROVED** [review]

**Pull request merged**